app/share.tsx

```diff
@@ -117,10 +123,12 @@ class Root extends React.Component<{}, IState> {
 
 		if (currentServer) {
 			await localAuthenticate(currentServer);
-			this.setState({ root: 'inside' });
+			// @ts-ignore
+			this.mounted ? this.setState({ root: 'inside' }) : (this.state.root = 'inside');
```

Comment:
You don't have to do this here, as `await localAuthenticate(currentServer)` already makes sure a render occurred.